app/src/main/java/com/android/sample/ui/activity/CreateActivity.kt

```diff
@@ -151,81 +155,94 @@ fun CreateActivityScreen(
                   },
               )
             }
-            Spacer(modifier = Modifier.height(8.dp))
+            Spacer(modifier = Modifier.height(STANDARD_PADDING.dp))
```

Comment:
I don't know how good of a practice would it be to directly store the .dp in the C file.

So that we no longer need to add .dp after each value.